pyproject.toml

```diff
@@ -149,7 +149,7 @@ towncrier = "^21.3.0"
 toml = "^0.10.0"
 pep440-version-utils = "^0.3.0"
 pydoc-markdown = "^3.10.3"
-pytest-timeout = "^2.1.0"
+pytest-timeout = "^1.4.2"
```

Comment:
Please revert this change (or explain why this is necessary)

Reply:
@ka-bu I think you reviewed the changes from last commit. [Comparing it with `main`, `pytest-timeout` should be left at `^1.4.2`](https://github.com/RasaHQ/rasa/pull/10944#discussion_r819591297).